### PR TITLE
fix(condo): DOMA-8515 qa review fixes

### PR DESCRIPTION
**Pull request by abshnko**

### CODE REVIEW

File: apps/condo/domains/meter/access/MeterReading.js [hunk 1]
```diff
@@ -76,6 +76,19 @@ async function canManageMeterReadings (args) {
...
 
+    if (operation === 'update' && itemId) {
+        const meterReading = await getByCondition('MeterReading', {
+            id: itemId,
+            deletedAt: null,
+        })
+        if (!meterReading) return false
+
+        const meterReadingOrganization = get(meterReading, 'organization')
+        if (!meterReadingOrganization) return false
```
Comment (pahaz) [inline apps/condo/domains/meter/access/MeterReading.js:87]: We also checks that organization is not undefined inside the checkPermissionsInEmployedOrRelatedOrganizations, but it’s ok to check it here too.

File: apps/condo/domains/meter/access/MeterReading.js [hunk 2]
```diff
@@ -76,6 +76,19 @@ async function canManageMeterReadings (args) {
         return await checkPermissionsInEmployedOrRelatedOrganizations(context, user, meterOrganization, 'canManageMeterReadings')
     }
 
+    if (operation === 'update' && itemId) {
```
Comment (pahaz) [inline apps/condo/domains/meter/access/MeterReading.js:79]: It looks strange if you change the access logic and don’t create or update any test

File: apps/condo/domains/meter/access/MeterReading.js [hunk 3]
```diff
@@ -76,6 +76,19 @@ async function canManageMeterReadings (args) {
...
+        const meterReading = await getByCondition('MeterReading', {
+            id: itemId,
+            deletedAt: null,
+        })
```
Comment (sitozzz) [inline apps/condo/domains/meter/access/MeterReading.js:83]: What real case does this check cover? This looks like quite a major operation (extra query each reading before update)

File: apps/condo/domains/meter/hooks/useMeterReadingFilters.tsx [hunk 4]
```diff
@@ -100,7 +102,7 @@ export function useMeterReadingFilters (): Array<FiltersMeta<MeterReadingWhereIn
                     },
                 },
             },
-            {
+            isPropertyMeter ? null : {
```
Comment (sitozzz) [inline apps/condo/domains/meter/hooks/useMeterReadingFilters.tsx:105]: This and the following expressions can be combined into one check, right?

File: apps/condo/domains/meter/access/MeterReading.js (hunk 2 above)
Comment (abshnko) [reply apps/condo/domains/meter/access/MeterReading.js]: added tests for update logic

File: apps/condo/domains/meter/components/TabContent/PropertyMeter.tsx [hunk 5]
```diff
@@ -187,30 +186,18 @@ const PropertyMetersTableContent: React.FC<PropertyMetersTableContentProps> = ({
...
-                                <MetersImportWrapper
-                                    key='import'
-                                    accessCheck={canManageMeters}
-                                    onFinish={refetch}
-                                />
-                            ),
-                        ]}
-                    />
+                    <ActionBar actions={[
+                        canManageMeters && (
```
Comment (nomerdvadcatpyat) [inline apps/condo/domains/meter/components/TabContent/PropertyMeter.tsx:190]: mb hide action bar when no canManageMeters?

File: apps/condo/domains/meter/hooks/useMeterReadingFilters.tsx [hunk 6]
```diff
@@ -115,7 +117,7 @@ export function useMeterReadingFilters (): Array<FiltersMeta<MeterReadingWhereIn
...
```
Comment (nomerdvadcatpyat) [inline apps/condo/domains/meter/hooks/useMeterReadingFilters.tsx:120]: it's better to made separate filters/columns hooks for separate pages or tabs

File: apps/condo/lang/en/en.json [hunk 7]
```diff
@@ -2347,7 +2350,7 @@
   "tasks.MeterReadingsImportTask.progress.title": "Import meter readings",
   "tasks.MeterReadingsImportTask.progress.description.preparing": "Preparing to import meter readings",
   "tasks.MeterReadingsImportTask.progress.description.processing": "Meter readings processed: {imported} of {total}",
-  "tasks.MeterReadingsImportTask.progress.description.completed": "File has been created",
+  "tasks.MeterReadingsImportTask.progress.description.completed": "File has been created, please refresh the page",
```
Comment (nomerdvadcatpyat) [inline apps/condo/lang/en/en.json:2353]: why does user need to update pages? in other imports we refetch data after import completed

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/meter/components/TabContent/PropertyMeter.tsx (hunk 5 above)
Comment (abshnko) [reply apps/condo/domains/meter/components/TabContent/PropertyMeter.tsx:190]: yes, fixed

File: apps/condo/lang/en/en.json (hunk 7 above)
Comment (abshnko) [reply apps/condo/lang/en/en.json:2353]: temporary solution because of meters not refetching

**Pull request merged**